R/mng01.R

```diff
@@ -74,7 +78,6 @@ mng01_main <- function(adam_db,
   assert_subset(table, c("n", center_fun_choice, interval_fun_choice))
 
   df <- adam_db[[dataset]]
-  line_col <- unlist(line_col)
```

Comment:
why unlist?

Reply:
I think we were accepting list before (now we are asserting for character) and we forgot to remove that bit.